### PR TITLE
Issue 875: Over Verbose Princess

### PR DESCRIPTION
**Pull request by dericpage**
Found the place where the chat wasn't sending the proper verbosity level for the message.

### CODE REVIEW

File: megamek/src/megamek/client/bot/princess/PathRanker.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public PathRanker(Princess princess) {
         owner = princess;
     }
 
-    private Map<MovePath.Key, Double> pathSuccessProbabilities = new HashMap<MovePath.Key, Double>();
+    private Map<MovePath.Key, Double> pathSuccessProbabilities = new HashMap<>();
```
Comment (Dylan-M) [inline megamek/src/megamek/client/bot/princess/PathRanker.java]: This change doesn't seem correct. You've removed the strict typing in half of the statement for no reason that I can see...

File: megamek/src/megamek/common/logging/LogLevel.java [hunk 2]
```diff
@@ -49,6 +49,10 @@ public int toInt() {
         return level.toInt();
     }
 
+    public boolean willLog(final LogLevel logLevel) {
```
Comment (Dylan-M) [inline megamek/src/megamek/common/logging/LogLevel.java]: This method is much better than what you were doing before. I like it. Nice work.

File: megamek/src/megamek/client/bot/princess/PathRanker.java (hunk 1 above)
Comment (dericpage) [reply megamek/src/megamek/client/bot/princess/PathRanker.java]: It's standard Java 7+ convention to type inference in place of parameterized types:
https://docs.oracle.com/javase/tutorial/java/generics/types.html#diamond

Comment (Dylan-M) [reply megamek/src/megamek/client/bot/princess/PathRanker.java]: I had forgotten about that change, and was having nightmares about the hundreds of places I've added parameterized types when Eclipse was throwing over 2000 warnings for them a few years ago. I'll approve the PR ;)

**Pull request merged**